docs/docs/extras/contribs.rst

```diff
@@ -0,0 +1,17 @@
+Community tools
+===============
+
+Utility scripts
+---------------
+
+Some utility scripts have been developped by isso users:
```

Comment:
developed rather than developped

Could you add a brief overview of the scripts, i.e. a single line per script?